docs/api/world.md

```diff
@@ -1,175 +1,358 @@
 # World
+A World contains entities which have components. The World is queryable and can be used to get entities with a specific set of components and to perform different kinds of operations on them.
 
-A World contains entities which have components. The World is queryable and can be used to get entities with a specific set of components.
+## Functions
+### new
+Creates a new world (necessary step to utilize the methods of the class)
 
-# Functions
-
-## new()
+`World` utilizes a class, meaning JECS allows you to create multiple worlds.
 ```luau
 function World.new(): World
 ```
-Creates a new world.
 
 Example:
-::: code-group
 
+::: code-group
 ```luau [luau]
 local world = jecs.World.new()
+local myOtherWorld = jecs.World.new()
 ```
 
 ```ts [typescript]
 import { World } from "@rbxts/jecs";
 
 const world = new World();
+const myOtherWorld = new World();
 ```
-
 :::
 
-## entity()
+## Methods
+### entity
+Creates a new entity.
 ```luau
-function World:entity(): Entity -- The new entit.
+function World:entity(): Entity
 ```
-Creates a new entity.
 
 Example:
-::: code-group
 
+::: code-group
 ```luau [luau]
 local entity = world:entity()
 ```
 
 ```ts [typescript]
 const entity = world.entity();
 ```
-
 :::
 
-## component()
+### component
+Creates a new component. Do note components are entities as well, meaning JECS allows you to add other components onto them.
+
+These are meant to be added onto other entities through `add` and `set`
 ```luau
 function World:component<T>(): Entity<T> -- The new componen.
 ```
-Creates a new component.
 
 Example:
-::: code-group
 
+::: code-group
 ```luau [luau]
-local Health = world:component() :: jecs.Entity<number>
+local Health = world:component() :: jecs.Entity<number> -- Typecasting this will allow us to know what kind of data the component holds!
 ```
 
 ```ts [typescript]
 const Health = world.component<number>();
 ```
 :::
 
-::: info
-You should use this when creating components.
-
-For example, a Health type should be created using this.
-:::
-
-## get()
+### get
+Returns the data present in the component that was set in the entity. Will return nil if the component was a tag or is not present.
 ```luau
 function World:get<T>(
     entity: Entity, -- The entity
     id: Entity<T> -- The component ID to fetch
-): T
+): T?
 ```
-Returns the data for the component data the corresponding entity, nil if entity does not have the ID or was a tag.
 
-## has()
+Example:
+
+::: code-group
+```luau [luau]
+local Health = world:component() :: jecs.Entity<number>
+
+local Entity = world:entity()
+world:set(Entity, Health, 100)
+
+local EntityWithNoHealth = world:entity()
+
+print(world:get(Entity, Health)) -- Outputs 100
+print(world:get(EntityWithNoHealth, Health)) -- Outputs nil

```

Comment:
This is redundant because it is obvious that if you do not set the data, it will just output nil. Examples should be less busy and convey the most important information and let the user figure out the semantics.